standalone/src/test/java/io/pravega/local/AuthEnabledInProcPravegaClusterTest.java

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.local;
+
+import io.pravega.client.ClientConfig;
+import io.pravega.client.ClientFactory;
+import io.pravega.client.admin.ReaderGroupManager;
+import io.pravega.client.admin.StreamManager;
+
+import io.pravega.client.stream.EventStreamWriter;
+import io.pravega.client.stream.ReinitializationRequiredException;
+import io.pravega.client.stream.ScalingPolicy;
+import io.pravega.client.stream.StreamConfiguration;
+import io.pravega.client.stream.impl.JavaSerializer;
+import io.pravega.client.stream.EventWriterConfig;
+import io.pravega.client.stream.ReaderGroupConfig;
+import io.pravega.client.stream.Stream;
+import io.pravega.client.stream.EventStreamReader;
+import io.pravega.client.stream.ReaderConfig;
+import io.pravega.client.stream.impl.DefaultCredentials;
+import io.pravega.test.common.AssertExtensions;
+import java.net.URI;
+import java.util.UUID;
+
+import lombok.Cleanup;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Integration tests for auth enabled in-process standalone cluster.
+ */
+@Slf4j
+public class AuthEnabledInProcPravegaClusterTest extends InProcPravegaClusterTest {
+    @Before
+    @Override
+    public void setUp() throws Exception {
+        this.authEnabled = true;
+        this.tlsEnabled = false;
+        super.setUp();
+    }
+
+    @Override
+    public void createTestStream() {
+        // his test from the base is unnecessary in this class as that and more is already being
+        // tested in another test method.
+    }
+
+    // Note: Strictly speaking, this test is really an "integration test" and is a little
```

Comment:
Why don't you use proper Javadoc for methods?

Reply:
Yeah, you are right. Will try to use them more frequently.